tests/e2e/e2e_pod_client_server_test.go

```diff
@@ -0,0 +1,132 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("1 Client pod -> 1 Server pod test", func() {
```

Comment:
Could you describe in comments the things that happen in this test.

Reply:
Will do.